src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -112,6 +112,12 @@
       "defaultValue": "true",
       "description": "Whether or not to use Reactive extensions"
     },
+    "cpm": {
+      "type": "parameter",
+      "dataType": "bool",
+      "defaultValue": "false",
+      "description": "Whether or not to use Central Package Management - package versions are via central Directory.Packages.props file"
+    },
```

Comment:
As discussed, make sure to add a CI validation with this flag enabled

Reply:
done